runtime/src/main/java/io/novafoundation/nova/runtime/storage/source/query/StorageQueryContext.kt

```diff
@@ -49,7 +49,8 @@ interface StorageQueryContext {
     suspend fun <K, V> StorageEntry.entries(
         vararg prefixArgs: Any?,
         keyExtractor: (StorageKeyComponents) -> K,
-        binding: DynamicInstanceBinderWithKey<K, V>
+        binding: DynamicInstanceBinderWithKey<K, V>,
+        onDecodeException: (Exception) -> Unit = { throw it }
```

Comment:
I think its more useful to have
`recover: (decodeException: Exception, rawValue: String) -> T?`